android-sdk/src/main/java/com/mobilecoin/lib/MobileCoinAccountClient.java

```diff
@@ -0,0 +1,86 @@
+package com.mobilecoin.lib;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import com.mobilecoin.lib.exceptions.AttestationException;
+import com.mobilecoin.lib.exceptions.FogReportException;
+import com.mobilecoin.lib.exceptions.InsufficientFundsException;
+import com.mobilecoin.lib.exceptions.InvalidFogResponse;
+import com.mobilecoin.lib.exceptions.InvalidTransactionException;
+import com.mobilecoin.lib.exceptions.NetworkException;
+import com.mobilecoin.lib.exceptions.TransactionBuilderException;
+import java.math.BigInteger;
+import java.util.concurrent.TimeoutException;
+
+/**
+ * Handles interactions with the user's MobileCoin account.
+ *
+ * <p>>Makes requests to the MobileCoin blockchain to provide information regarding the account.
+ */
+public interface MobileCoinAccountClient {
+
+  /**
+   * Fetches the latest account snapshot.
+   */
+  @NonNull
+  AccountSnapshot getAccountSnapshot() throws NetworkException,
+      InvalidFogResponse, AttestationException;
+
+  /**
+   * Creates an account snapshot for the provided block index.
+   *
+   * @param blockIndex is the index of the last block to include TxOuts from
+   * @return {@link AccountSnapshot} or {@code null} if the {@code blockIndex} is higher than
+   * what fog currently at and not equals to {@link UnsignedLong#MAX_VALUE}
+   */
+  @Nullable
+  AccountSnapshot getAccountSnapshot(UnsignedLong blockIndex) throws NetworkException,
+      InvalidFogResponse, AttestationException
+
+  /**
+   * Retrieves {@code AccountKey}'s balance.
+   */
+  @NonNull
+  Balance getBalance() throws InvalidFogResponse, NetworkException, AttestationException;
+
+  /**
+   * Returns whether the defragmentation is required on the active account in order to send the
+   * specified amount
+   */
+  boolean requiresDefragmentation(@NonNull BigInteger amountToSend)
+      throws NetworkException, InvalidFogResponse, AttestationException,
+      InsufficientFundsException;
+
+  /**
+   * Defragments the user's account.
```

Comment:
This is a Github text parsing error. The comments are syntactically correct and Android Studio displays it without any errors.